### PR TITLE
Tucker/job 71  changelog but in new sub project

### PR DESCRIPTION
**Pull request by dusktreader**
#### What
Added a CHANGELOG.

Also pinned project version to 0.0.1 until first release. This will allow us to import the release script from jobbergate-cli and have the first proper release be versioned 0.1.0

#### Why
We needed a CHANGELOG

`Task`: https://app.clickup.com/t/18022949/JOB-71

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

File: jobbergate-api/pyproject.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ authors = [
 name = "jobbergateapi2"
 description = "jobbergate-api"
 license = "MIT"
-version = "0.1.0"
+version = "0.0.1"
```
Comment (heitorPB) [inline jobbergate-api/pyproject.toml:8]: ??????

Comment (dusktreader) [reply jobbergate-api/pyproject.toml:8]: I did this so the first time we run the release script, we can bump the minor version. It's still unreleased, so this doesn't mean that much at this point. Poetry sets the initial version to 0.1.0 by default, but I want that to be the first version we actually release.

If it's too weird, I can just make 0.2.0 the first released version

Comment (heitorPB) [reply jobbergate-api/pyproject.toml:8]: Let's not roll back versions, this can cause a lot of confusion.

Comment (dusktreader) [reply jobbergate-api/pyproject.toml:8]: It's unreleased. It's not really a rollback.

**Review by heitorPB — APPROVED** [review]

**Review by jamesbeedy — APPROVED** [review]
👍

**Pull request merged**